docs/getting-started/flashing.md

```diff
@@ -0,0 +1,123 @@
+# Flashing Zarhus OS
+
+There are many ways an OS can be flashed and booted on embedded platform, but
+the most common one is using an SD card. This guide will describe SD card
+flashing only, because all other ways to flash the image are highly
+target-specific and are described [per-target](../supported-targets/targets.md).
+
+## Flashing SD card
+
+This section demonstrates how to flash a Zarhus OS image on the SD card.
+
+### Prerequisites
+
+* Linux PC (tested on `Ubuntu 20.04 LTS`);
+* [bmaptool](https://source.tizen.org/documentation/reference/bmaptool)
+  installed:
+
+    ```bash
+    sudo apt install bmap-tools
+    ```
+
+* Zarhus OS image built according to [build guide](./building.md).
+
+!!! note
+
+    You can also use `bmap-tools` [from
+    GitHub](https://github.com/intel/bmap-tools) if it is not available in your
+    distro.
+
+### Flashing
+
+Find out your device name:
+
+```shell
+$ fdisk -l
+(...)
+Device     Boot  Start    End Sectors  Size Id Type
+/dev/sdx1  *      8192 131433  123242 60,2M  c W95 FAT32 (LBA)
+/dev/sdx2       139264 186667   47404 23,2M 83 Linux
+```
+
+!!! warning
+
+    In this case the device name is `/dev/sdx` **but be aware, in next steps
+    replace `/dev/sdx` with the right device name on your platform or else you
+    can damage your system!**
+
+From the directory you ran your image build, run command:
+
+```shell
+$ cd build/tmp/deploy/images/MACHINE_NAME
+$ sudo umount /dev/sdx*
+$ sudo bmaptool copy --bmap IMAGE_NAME-IMAGE_TYPE-MACHINE_NAME.rootfs.wic.bmap IMAGE_NAME-IMAGE_TYPE-MACHINE_NAME.rootfs.wic.gz /dev/sdx
```

Comment:
```suggestion
$ sudo bmaptool copy IMAGE_NAME-IMAGE_TYPE-MACHINE_NAME.rootfs.wic.gz /dev/sdx
```

bmaptool will automatically look for file with the same name but with `.bmap` ending instead of `.gz`

Reply:
Changed here eaa2ab81ced422f758e8d85075b4cceb2aaad608.